src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -1278,7 +1278,7 @@ void func_80A14018(EnGo* this, PlayState* play) {
     s32 phi_v0 = ENGO_GET_70(&this->actor) % 6;
 
     if (phi_v0 < 4) {
-        phi_v0 = ((gSaveContext.eventInf[2] & 0xF) + phi_v0) % 4;
+        phi_v0 = ((gSaveContext.eventInf[EVENTINF_02] & 0xF) + phi_v0) % 4;
```

Comment:
Same here.

Reply:
I'm fairly sure that this isn't bitpacked either. There's no shifting going on here.

Reply:
This looks to be using `EVENTINF_20` through `EVENTINF_23`.

Reply:
To be clear, I'm not saying the actual access to the eventInf array here is bitpacked, but the `EVENTINF_02` isn't meant to be used in this way.

This specific line of code is accessing the values covered by the following macros
```c
#define EVENTINF_20 0x20
#define EVENTINF_21 0x21
#define EVENTINF_22 0x22
#define EVENTINF_23 0x23
```

It is using a range of the eventInf flags to store data that are not flag, like the Doggy racetrack does to store the textId in the weekEventReg, or the horse race flags